Update nudge.md (#64)
transferwise/neptune: src/components/nudge.md

Nudges direct users to another view to complete an action. Because the content they contain is optional, nudges are always dismissible.

> The whole component is tappable — it triggers the link. But tapping the area around the close icon dismisses the nudge. 

Author: ```suggestion
Tapping anywhere on the component triggers the link — except for the area around the close icon, which dismisses the nudge.
```
